src/components/include/resumption/last_state.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2018, Ford Motor Company
```

Comment:
@YarikMamykin 2019

Reply:
@AKalinich-Luxoft fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/61f8fecf23bd114017777ce8d90cdb0fb952edf1